client/me/purchases/manage-purchase/style.scss

```
@@ -434,3 +443,9 @@
 		margin-top: 0;
 	}
 }
+
+.manage-purchase__no-payment-method {
+	svg {
+		flex-shrink: 0;
+	}
+}
```

This feels like it should be part of the other `.manage-purchase__no-payment-method` section being added in this PR. If this is supposed to stay separate from that one, then we should use a single selector.
```suggestion

.manage-purchase__no-payment-method svg {
	flex-shrink: 0;
}
```

---

They've been grouped.